roles/comanage-registry/tasks/install.yml

```diff
@@ -75,8 +67,8 @@
     group: "{{ comanage_webserver_group }}"
     recurse: yes
   become: yes
-  when: GCOmanageCheck.changed
 
+# IMPORTANT:The symlink should be owned by web user or else it will break the two controls from above
```

Comment:
This comment can be removed since the symlink does not need to be owned by the web user.